client/configuration/webpack/webpack.config.prod.js

```diff
@@ -99,6 +99,9 @@ const prodConfig = {
         NODE_ENV: "production",
       }),
     }),
+    new webpack.ProvidePlugin({
+      ReadableStream: ["web-streams-polyfill/polyfill", "ReadableStream"],
```

Comment:
just curious, since Explorer only supports modern browsers, why do we need to polyfill `ReadableStream`? Thank you!

Reply:
it's a breaking change in webpack 5. webpack < 5 used to include core node polyfills by default.

`ReadableStream` is required by `undici` - an HTTP client which is a dep for`@chromatic-com/playwright` latest version.

However, I initially attributed the fix to the above instead on the node version 18 update  (which was part of the same commit), but it was the latter that did the trick, and this is apparently leftover from debugging attempt.

Just pushed a commit to remove and also to update node to 20 as discussed

Reply:
Ahh TIL! Thanks so much for the background info 💡 !!